readthedocs/profiles/views.py

```diff
@@ -421,5 +422,11 @@ def post(self, request, *args, **kwargs):
                         "project_slug": project.slug,
                     },
                 )
+            count += 1
+
+        if count > 0:
+            messages.success(request, _(f"Successfully migrated {count} project(s)."))
+        else:
+            messages.info(request, _("No projects were migrated."))
```

Comment:
This seems like an error? We should definitely communicate a bit more about next steps if this happens.

Reply:
I understand this is because the user didn't click on "Migrate" button for any of the projects. If that's the case, it seems it's not an error.